ppr-ui/src/views/dashboard.vue

```diff
@@ -51,6 +91,42 @@ export default class Dashboard extends Vue {
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
+  private get searchTime (): string {
+    // return formatted date
+    const searchResult = this.getSearchResults
+    if (searchResult) {
+      const searchDate = new Date(searchResult.searchDateTime)
+      let timezone = ''
+      if ((searchDate.toString()).includes('Pacific')) timezone = 'Pacific Time'
+      // format datetime -- have to put in zeros manually when needed
+      let hour = `0${searchDate.getHours()}`
+      let min = `0${searchDate.getMinutes()}`
+      let sec = `0${searchDate.getSeconds()}`
+      if (hour.length > 2) hour = hour.slice(1)
+      if (min.length > 2) min = min.slice(1)
+      if (sec.length > 2) sec = sec.slice(1)
+      const datetime = `${hour}:${min}:${sec}`
+      return 'saved ' + moment(searchDate).format('MMMM D, Y') + ` ${datetime} ${timezone}`
+    }
+    return ''
+  }
```

Comment:
If this is something you might re-use i'd probably make it common within a mixin (or perhaps composition api?! not sure how works yet), and just pass the `getSearchResults` to it.

Reply:
Yes good point I will update this too